src/components/Inputs/CustomNumberField.js

```diff
@@ -16,14 +16,17 @@ const CustomNumberField = ({
   decimalScale = 2,
   onClear,
   hidden = false,
+  unClearable = false,
   error,
   helperText,
   hasBorder = true,
   maxLength = 1000,
   thousandSeparator = ',',
-  min = '',
-  max = '',
+  min,
+  max,
```

Comment:
why did you remove the =''? it was here for the case where we dont put min and max

Reply:
we can gave it directly from the component  example : min={1}